tortoisebeacon/tortoise_beacon.go

```diff
@@ -222,6 +199,7 @@ func (tb *TortoiseBeacon) IsClosed() bool {
 }
 
 // GetBeacon returns a Tortoise Beacon value as []byte for a certain epoch or an error if it doesn't exist.
+// TODO(nkryuchkov): consider not using (using DB instead)
```

Comment:
are beacons not persisted now?

Reply:
@antonlerner They are stored in the `beacons` map and in the tortoise beacon database. But I think we can get rid of the `beacons` map and use the database only